backend/app/main.py

```
@@ -0,0 +1,74 @@
+from contextlib import asynccontextmanager
+from typing import Annotated
+
+from fastapi import Depends, FastAPI
+
+from app.api.v1.api import api_router
+from app.core.config import settings
+from app.db.base import Base
+from app.db.session import engine
+
+# Import models to ensure they are registered with Base
+from app.models import ChatHistory, NewsSubscription, SmartDevice, User  # noqa: F401
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)